### ISSUE TITLE
No Method to Visually Display Speaker Photo Alt Text Descriptions

### ISSUE THREAD

**Issue by dzoladz** [code4lib/2022.code4lib.org#5]
This issue is dependent upon:
1. the completion of issue #4, as this would be enhancement to provide a function to make the alt text visible via user interaction.
2. the existence of a method to obtain alt text descriptions from accepted conference speakers

**Comment by dzoladz** [comment]
> A question for this issue is how we should handle missing photos and missing alt-text. I can bring this to the accessibility committee when we meet too.

- For missing alt text, should we default to the speaker name, or an empty alt attribute?
- For a missing photo, should we add alt-text to the "no pic" photos? e.g. the photos of a pug with 404 text.

@kaitlinnewson Was this ever discussed with the accessibility committee?

**Comment by metageeky** [comment]
@dzoladz  My code already handles this situation. 

If there is no speaker image or the image is one of the nopic ones, the alt text is "Random placeholder image for [speaker name]".

If there is a speaker image but no provided alt text, the alt text is "Profile image for [speaker name]"

**Comment by metageeky** [comment]
Also, if the speaker provided alt text is present, the alt text is shown automatically when a speaker info is toggled. It only shows this when it's speaker provided. I decided on this approach as it is the cleanest, simplest approach that doesn't require adding other javascript functionality.
